runtime.md

```diff
@@ -16,6 +16,7 @@ There is no requirement that it be unique across hosts.
 The ID is provided in the state because hooks will be executed with the state as the payload.
 This allows the hooks to perform cleanup and teardown logic after the runtime destroys its own state.
 * **`pid`**: (int) is the ID of the main process within the container, as seen by the host.
+The pid MAY be excluded or set to a value indicating empty or null if it is not seen by the host.
```

Comment:
I'd rather phrase this as:

> * **`pid`** (int, optional) is the ID ….
>   This property is REQUIRED on the following platforms: Linux, …

although that's tied up in [specifying platforms for VM-based containers][1].

[1]: https://github.com/opencontainers/runtime-spec/pull/405#issuecomment-220721379